pkg/depends_on.go

```diff
@@ -0,0 +1,35 @@
+package pkg
+
+import "fmt"
+
+func buildInstallOrder(svcs []Service) []string {
+	graph := Graph{nodes: []*Node{}}
+
+	nodeMap := map[string]*Node{}
+	for _, s := range svcs {
+		n := &Node{Name: s.Name}
+		nodeMap[s.Name] = n
+		graph.nodes = append(graph.nodes, n)
+	}
+
+	for _, s := range svcs {
+		for _, d := range s.DependsOn {
+			nodeMap[s.Name].Edges = append(nodeMap[s.Name].Edges, nodeMap[d])
+		}
+	}
+
+	resolved := &Graph{}
+	unresolved := &Graph{}
+	for _, g := range graph.nodes {
+		resolve(g, resolved, unresolved)
+	}
+
+	fmt.Printf("Installation order:\n")
```

Comment:
should this be removed?

Reply:
I am not sure, I think for the first few revisions it could be useful to have the output in the logs of users

Reply:
I've updated the name of this output and made it a log statement.